mentors/README.md

```diff
@@ -1,76 +1,73 @@
-## Classification Data Pipeline Info
+## Mentor Pipeline
 ---------------
+
+Use these tools to create/update a mentor for Mentorpal.
+
 ### Quick Start
 
+#### Required Setup (session videos and timstamps)
+TODO: needs to download session videos and timestamp files automatically but for now...
+Download session videos and timestamps and place them under `./data/mentors/{mentor_id}/build/recordings` eg.
```

Comment:
maybe include a link where these can be downloaded from? or not needed since they'll eventually be done automatically?

Reply:
there's no consistent link. It's where ever they put the latest one for now